.github/workflows/python-app.yml

```diff
@@ -174,3 +173,67 @@ jobs:
       with:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         branch: gh-pages
+
+  build-executables:
+    runs-on: ubuntu-latest
+    needs: [full_test]
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository with full history
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Number of commits to fetch. 0 indicates all history for all branches and tags.
+
+      - name: Build Docker image
+        uses: docker/build-push-action@v2.7.0
+        with:
+          context: .
+          push: false
+          tags: pymchelper
+
+      - name: Generate single-file executables 1
+        run: |
+          docker run -v `pwd`/dist:/app/dist pymchelper:latest pyinstaller --add-data 'pymchelper/VERSION:pymchelper' -F pymchelper/utils/mcscripter.py
+
+      - name: Generate single-file executables 2
+        run: |
+          docker run -v `pwd`/dist:/app/dist pymchelper:latest pyinstaller --add-data 'pymchelper/VERSION:pymchelper' -F pymchelper/utils/pld2sobp.py
+
+      - name: Generate single-file executables 3
+        run: |
+          docker run -v `pwd`/dist:/app/dist pymchelper:latest pyinstaller --add-data 'pymchelper/VERSION:pymchelper' -F pymchelper/utils/runmc.py
+
+      - name: Generate single-file executables 4
+        run: |
+          docker run -v `pwd`/dist:/app/dist pymchelper:latest pyinstaller --add-data 'pymchelper/VERSION:pymchelper' -F pymchelper/run.py
```

Comment:
Instead of numbers in names you could use actual names of `mcscripter`, `runmc`, ...

Reply:
Fixed in 2a8869e